spec/requests/api/projects_spec.rb

```diff
@@ -105,7 +105,7 @@
     end
 
     it "ignores stuff before start_time" do
-      after_everything = [project.updated_at, dependent_project.updated_at].max + 1
+      after_everything = [project.reload.updated_at, dependent_project.reload.updated_at].max + 1
```

Comment:
It looks like this should already happening in the `before` block on line 19,20. Is there something happening between that block and this spec that the reload is fixing?

Reply:
Ah good catch. Now I see what happened. The failing test returned this third project that we weren't considering in the `after_everything` calculation.